py/surveyqa/nightly.py

```diff
@@ -351,23 +386,48 @@ def makeplots(night, exposures, tiles, outdir):
         </div>
     """.format(night)
 
+    template += """
+        <ul>
+          <li><a href={}>First</a></li>
+          <li><a href={}>Previous</a></li>
+          <li><a href={}>Summary Page</a></li>
+          <li><a href={}>Next</a></li>
+          <li><a href={}>Last</a></li>
+        </ul>             
+    """.format(first_str, prev_str, summary_str, next_str, last_str)
+    
     template += """
         <div class="flex-container">
-                <div class="column side"></div>          
-                <div class="column middle">
-                    <div class="flex-container">
-                        <div>{{ skypathplot_script }} {{ skypathplot_div}}</div>
-                        <div>{{ exptypecounts_script }} {{ exptypecounts_div }}</div>
-                    </div>    
+            <div class="column side"></div>    
+            <div class="column middle">
+                <div class="flex-container">
+                    <div>{{ skypathplot_script }} {{ skypathplot_div}}</div>
+                    <div>{{ exptypecounts_script }} {{ exptypecounts_div }}</div>
+                </div>    
                     
+<<<<<<< HEAD
+                <div class="flex-container">
+                    <div>{{ timeseries_script }} {{ timeseries_div }}</div>
+                    <div> NIGHT VS. SUMMARY HISTOGRAMS HERE </div>
+                </div> 
+
+                <div class="flex-container">{{ table_script }}{{ table_div }}</div>
+            </div>     
+            <div class="column side"></div>        
+        </div>          
+=======
                     <div class="flex-container">
                         <div>{{ timeseries_script }} {{ timeseries_div }}</div>
                     </div> 
 
                     <div class="flex-container">{{ table_script }}{{ table_div }}</div>     
                 </div>
+    """
+    template += """
                 <div class="column side"></div>
+                    <a href={}>Next</a>
             </div>
+>>>>>>> 2740d584a08dd6f2e9054ea0c4833525bd611a9e
```

Comment:
Another merge conflict to resolve.  This one is a little less clear to me which version is to be kept.